src/doc/en/developer/coding_basics.rst

```diff
@@ -1039,14 +1039,9 @@ written.
     The :ref:`doctest fixer <section-fixdoctests-optional-needs>` uses
     tab stops at columns 48, 56, 64, ... for these tags.
 
-- **Python3 print:** Python3 syntax for print must be used in Sage
-  code and doctests. If you use an old-style print in doctests, it
-  will raise a SyntaxError::
-
-      sage: print "not like that"
-      Traceback (most recent call last):
-      ...
-      SyntaxError: ...
+- **Python3 print:** In SageMath code and doctests, adhere to Python 3
+  syntax for the print function. Use parentheses as follows:::
```

Comment:
why do you need `:::` there? Maybe `::` should do?

Reply:
I removed the `:` from end of the line.

Reply:
I would be more in favour of removing the sentence and the doctest completely, don't you agree @dimpase?  There is already so much to read and I don't think that anybody will be assuming python 2 syntax nowadays.

Reply:
good idea. It's just very basic python.

Reply:
@mantepse & @dimpase so should i move forward with removing the whole print sentence and doctest?

Reply:
yes please